### ISSUE TITLE
Doesn't work on Iris

### ISSUE THREAD

**Issue by ruvaldak** [ruvaldak/VanillaBean-shaders#10]
While my original intent was for this to work with Iris, it currently does not. It loads, but everything with the exception of water and clouds is solid black. While we could just wait for Iris to add support for.. whatever is in this that it doesn't support, I'd much rather find out what is breaking this on Iris and find a way to remedy it if possible.

**Comment by ruvaldak** [comment]
```
#version 120

#extension GL_ARB_shader_texture_lod : enable

uniform mat4 gbufferProjectionInverse;

#ifdef GLSLANG
#extension GL_GOOGLE_include_directive : enable
#endif

varying vec2 texCoord;

uniform sampler2D colortex0;
uniform sampler2D colortex5;
uniform sampler2D depthtex0;
uniform float viewWidth;
uniform float viewHeight;

#include "settings.glsl"

void main()
{
    #ifdef SSAO
    //fast simple small bilateral blur by gri573
    int size = AO_BLUR_SIZE;
    float clarity = AO_BLUR_CLARITY;
    float col = texture2D(colortex5, texCoord).r;
    float weight = AO_BLUR_WEIGHT;

    float col1 = 0.0;
    for(int i = 0; i < size; i++) {
        for(int j = 0; j < size; j++) {
            float col0 = texture2D(colortex5, texCoord + (vec2(i, j) - 0.5 * vec2(size)) / vec2(viewWidth, viewHeight)).r;
            float weight0 = max(1.0 - clarity * length(col0 - col), 0.00001);
            col1 += weight0 * col0;
            weight += weight0;
        }
    }
    col = col1 / weight;

    /*DRAWBUFFERS:0*/
    gl_FragData[0] = vec4(texture2D(colortex0, texCoord).rgb * col,1.0);
    #endif
}
```

Something about this being my deferred1.fsh is leading to Iris being incapable of rendering VanillaBean correctly.

**Comment by ruvaldak** [comment]
fixed with e7bc1c2ea23b5b867af17b8ba8d1dcfe9c8a93e3
